.github/workflows/nodejs.yml

```
@@ -53,3 +53,7 @@ jobs:
         uses: coverallsapp/github-action@v1.1.2
         with:
             github-token: ${{ github.token }}
+      - name:
```

```suggestion
      - name: Build TypeDoc & SassDoc
```
Wondering if we should run this step for all node versions or just one (like coveralls above)